src/index.js

```diff
@@ -3,9 +3,10 @@ const path = require("path"),
 	worker = path.join(__dirname, "worker.js"),
 	events = /^(error|message)$/,
 	defaultPorts = {inspect: 9229, debug: 5858};
+let range = {min: 1, max: 300};
 
 class Worker {
-	constructor (arg, args = undefined, options = {cwd: process.cwd()}) {
+	constructor (arg, args = [], options = {cwd: process.cwd()}) {
```

Comment:
yes, but if it is undefined, `options.execArgv` is ignored and `process.execArgv` is used instead in fork. I don't know if that is inteded or just a side effect of some merging of arguments. This wasn't a problem earlier because of the bug wich was fixed in the first commit.